app/views/devise/registrations/new.html.erb

```diff
@@ -25,9 +27,11 @@
           </br>
           <%= f.submit "Sign up",:class => "sign-in" %>
         </div>
-        <div class="col-xs-10 col-xs-offset-2">
+        <div class="col-xs-10">
           </br>
-          <%= link_to "I already have an account", new_user_session_path,:class => "black" %>
+          <center>
+          <%= link_to "I already have an account", new_user_session_path,:class => "soft-black" %>
```

Comment:
Remember the spaces after the commas

Also, use the new hash syntax: `class: 'soft-black'`